### ISSUE TITLE
Error in upload

### ISSUE THREAD

**Issue by aviks** [JuliaCon/juliacon-webapp#37]
Reported by an user via email: 

_"I usually use Safari (13.1.1 on Mac OS 10.15.5 Catalina), but I also tried Chrome (83.0.4103.116) as well as Firefox (78.0.1)."_
```
upload.js:1 S3 upload failed: TypeError: Failed to execute 'setRequestHeader' on 'XMLHttpRequest': Value is not a valid ByteString.
    at upload.js:1
    at Object.c [as forEach] (upload.js:1)
    at upload.js:1
    at new Promise (<anonymous>)
    at e.exports (upload.js:1)
    at UnBK.e.exports (upload.js:1)
(anonymous) @ upload.js:1
upload.js:1 [ERROR] 05:15.753 axios-http-handler TypeError: Failed to execute 'setRequestHeader' on 'XMLHttpRequest': Value is not a valid ByteString.
    at upload.js:1
    at Object.c [as forEach] (upload.js:1)
    at upload.js:1
    at new Promise (<anonymous>)
    at e.exports (upload.js:1)
    at UnBK.e.exports (upload.js:1)
e._log @ upload.js:1
e.error @ upload.js:1
[ … a lot of anonymous functions … ]
upload.js:1 S3 upload failed: TypeError: Failed to execute 'setRequestHeader' on 'XMLHttpRequest': Value is not a valid ByteString.
    at upload.js:1
    at Object.c [as forEach] (upload.js:1)
    at upload.js:1
    at new Promise (<anonymous>)
    at e.exports (upload.js:1)
    at UnBK.e.exports (upload.js:1)
```

**Comment by travigd** [comment]
I will not have time to look at this until Wednesday  most likely. Will try sooner if I have time!!

**Comment by aviks** [comment]
So a more helpful error message from Firefox says `axios-http-handler TypeError: XMLHttpRequest.setRequestHeader: Cannot convert argument 2 to ByteString because the character at index 19 has value 8211 which is greater than 255.`

So it turns out this user's talk title had an `endash`, ie, a `–` in the title. Removing it makes the error go away. 

The title is meant to be set as a S3 object metadata, uploaded as an HTTP header. Setting unicode values as an http header is what causes an issue. 

The fix is beyond my JS/TS skillset, but hopefully this analysis is useful.

**Comment by travigd** [comment]
Fixed in 2dd456d308889b014c8b8ca0cbe2a78b90c4973b
